src/main/java/com/pingcap/tikv/Snapshot.java

```diff
@@ -121,6 +121,22 @@ public ByteString get(ByteString key) {
     }
   }
 
+  /**
+   * Below is lower level API for env like Spark which already did key range split Perform handle
+   * scan
+   *
+   * @param dagRequest DAGRequest for coprocessor
+   * @param tasks      RegionTask of the coprocessor request to send
+   * @return Row iterator to iterate over resulting rows
+   */
+  public Iterator<Long> handleRead(TiDAGRequest dagRequest, List<RegionTask> tasks) {
```

Comment:
I think `readHandle` better. `handler` could be a verb means process read request.

Reply:
Agree.

Reply:
This naming originally aims to stay consistent with the method `tableRead` above, what about changing `handleRead` to `indexRead`?

Reply:
that is probably fine too.